src/maliput/routing/route.cc

```diff
@@ -359,5 +364,130 @@ int Route::FindDirectionTowardsLaneSRangeWithStraightPredecessor(const Route::La
   return kTowardsRight;
 }
 
+namespace {
+
+// Finds a RoadPosition by matching @p lane pointer.
```

Comment:
Add blank line separating method summary line from parameter descriptions. Ditto similar instances below.

Reply:
Fixed in [75feba2](https://github.com/maliput/maliput/pull/628/commits/75feba2d628971074be119f12ded545d35d9fd29).